lm_admin_guide.md

```diff
@@ -323,52 +323,99 @@ required by the `psycopg2` Python package.
 
 ## How to upgrade
 
-### Upgrade a Docker deployment
-Existing Dockerized deployments can be easily upgraded to a more recent LifeMonitor version by going through the following steps:
-1. stop `lm` service
-    ```bash
-    docker-compose stop lm
-    ``` 
-2. make a backup of LifeMonitor to a temp path within the `db` container
-    ```bash
-    docker-compose exec db /bin/bash -c "PGPASSWORD=\${POSTGRESQL_PASSWORD} pg_dump -U \${POSTGRESQL_USERNAME} \${POSTGRESQL_DATABASE} > /tmp/lifemonitor_backup.sql"
-    ```
-3. copy backup from the `db` container to a local path
+> **WARNING.** It is strongly recommended to make a full backup of the LifeMonitor database (see sections ["How to backup and restore"](#how-to-backup-and-restore)) before upgrading.
+
+##### Native installation
```

Comment:
Heading should be level 3, not 5. The same goes for other sub-headings in this section.